bootstrap.sh

```diff
@@ -69,14 +69,53 @@ cd google-cloud-java
 git add pom.xml
 git commit -am 'feat: create aggregator pom'
 
+num_modules="$(wc -l < ../../repos.txt)"
+#echo "${num_modules}"
+touch .release-please-manifest.json
+echo "{" >> .release-please-manifest.json
+
 # generate BOM of the artifacts in this repository
 bom_lines=""
 for bom_directory in $(find . -name 'google-*-bom' | sort); do
   repo_metadata="${bom_directory}/../.repo-metadata.json"
   pom_file="${bom_directory}/pom.xml"
   groupId_line=$(grep --max-count=1 'groupId' "${pom_file}")
   artifactId_line=$(grep --max-count=1 'artifactId' "${pom_file}")
+
+  # extracting module name
+  prefix="  <artifactId>"
+  suffix="-bom</artifactId>"
+  artifactName=${artifactId_line#"$prefix"}
+  artifactName=${artifactName%"$suffix"}
+  prefix="./"
+  suffix="/${artifactName}-bom"
+  module=${bom_directory#"$prefix"}
+  module=${module%"$suffix"}
+#  echo "the module name is ${module}"
+
   version_line=$(grep --max-count=1 'x-version-update' "${pom_file}")
+
+  #extracting module version
+  prefix="  <version>"
+  suffix="</version><!-- {x-version-update:${artifactName}:current} -->"
+  module_version=${version_line#"$prefix"}
+  module_version=${module_version%"$suffix"}
+#  echo "the module version is ${module_version}"
+
+  #concatenating module name and module version
+  rp_config_line=""\""${module}"\"": "\""${module_version}"\"""
+
+  #adding " , " where it's necessary
+  if [ $num_modules -gt 1 ]
+    then
```

Comment:
Put ; then and ; do on the same line as the if, for, or while. (go/shell-style#control-flow)